src/assets/icons/BxDraggable.tsx

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Hmm I think I made a pr to make this not throw an error already, am I mistaken here?

Reply:
i just copy pasted so you might be right here? i can remove